src/components/address-input/autosuggest.address.spec.js

```diff
@@ -180,103 +120,218 @@ describe('script: address-input', () => {
     beforeEach(async () => {
         await apiFaker.reset();
     });
+    describe('GIVEN: the component initializes', () => {
+        describe('WHEN: the component initializes without options', () => {
```

Comment:
Theres some repetition here. Im not sure if we have set rules around what should go in GIVEN and WHENs in script tests. This may need some discussion.